src/lib.rs

```diff
@@ -164,7 +164,7 @@ mod tests {
 
         setup_env_vars(region);
 
-        let aurora_client = AuroraClient::new(&config, None).await;
+        let aurora_client = AuroraClient::try_new(&config, None).await.unwrap();
```

Comment:
if the unwrap returns an error? the daemon will stop right? Not for now, but we may add a retry mechanism in the future or assign to some intern. @blaginin